tests/problem_model.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<model unit="millimeter" xmlns="http://schemas.microsoft.com/3dmanufacturing/core/2015/02" xml:lang="en-US">
+	<!-- this file uses Dutch style commas for the decimal-indicator, instead of international/US points -->
+    <!-- this should either be read or fail fast, but should definitely not just silently ignore the part after the comma-->
```

Comment:
I'm gonna replace ALL TEH TABS WITH SPACES! :smiling_imp:

Reply:
... oops. Guess I ran out of spaces, glad you could lend me some.

(Also for future reference: 'resolve' doesn't 'comment and resolve' but just deletes the comment.)